nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -1002,13 +1002,7 @@ in
       };
 
       mapHashBucketSize = mkOption {
-        type = types.nullOr (
-          types.enum [
-            32
-            64
-            128
-          ]
-        );
+        type = types.nullOr (types.ints.positive);
         default = null;
         description = ''
           Sets the bucket size for the map variables hash tables. Default
```

Comment:
Maybe we can add https://nginx.org/en/docs/hash.html to the description? Otherwise LGTM